### ISSUE TITLE
Refresh Schedules don't work for Creating Tables/Views in Query

### ISSUE THREAD

**Issue by brad-natelborg** [getredash/redash#4376]
### Issue Summary

I have several queries in Redash that only create new tables, nothing else.  So they don't return any tables of data.  That being said, when I try to set a schedule to refresh them, it does not work unless I have the query actually return some table of data.

To work around it currently, I have just a dummy query at the end that selects some data from the newly created table.

It would be nice if Redash didn't return an error when a query was only creating a table or view and allowed them to be scheduled to update automatically.  

### Steps to Reproduce

See above.  Create a query that only creates a table, but doesn't return any data.

### Technical details:

* Redash Version: 7.0.0
* Browser/OS: Chrome / Windows 10
* How did you install Redash:

**Comment by arikfr** [comment]
Yes, that's a known limitation. We used the error mechanism to signal to the UI that no results were returned. Obviously it's not a great solution 😬 Also this behavior depends on the data source you use.

When I think of it now, I'm not sure if there is anything that prevents us from returning an empty `data` object in case of such query + update the UI to show some friendlier message about no results being available.

It's not something we will address right now, but if someone wants to pick it up, happy to discuss details.
